libnavigation-core/src/test/java/com/mapbox/navigation/core/routerefresh/RouteRefreshPauseResumeIntegrationTest.kt

```diff
@@ -0,0 +1,31 @@
+package com.mapbox.navigation.core.routerefresh
+
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+@OptIn(ExperimentalCoroutinesApi::class, ExperimentalPreviewMapboxNavigationAPI::class)
+internal class RouteRefreshPauseResumeIntegrationTest : RouteRefreshIntegrationTest() {
```

Comment:
what do you think about testing the case, when resume called multiple times?

Reply:
It's covered in `PlannedRouteRefreshControllerTest`. I'm not sure whether adding an integration test too would make a lot of sense here.